src/app.js

```diff
@@ -312,6 +315,19 @@ class App extends PureComponent {
       <div className='app'>
         {this.renderOptions()}
         {this.renderLogo()}
+        <div>
```

Comment:
I know this is WIP, but just making note to follow the pattern and create a class method called `renderQRCode` and run it here.